cargo-dist/templates/ci/github_ci.yml.j2

```diff
@@ -135,6 +135,8 @@ jobs:
   upload-global-artifacts:
     needs: upload-local-artifacts
     runs-on: {{{ global_task.runner }}}
+    outputs:
+      releases: ${{ steps.upload-global-artifacts.outputs.releases }}
```

Comment:
Arguably this should be being done on the create-release task, but this is fine for now.

Reply:
Easy change to make!